helios-services/src/main/java/com/spotify/helios/agent/AgentConfig.java

```diff
@@ -369,4 +371,13 @@ public AgentConfig setConnectionPoolSize(final int connectionPoolSize) {
     this.connectionPoolSize = connectionPoolSize;
     return this;
   }
+
+  public File getGoogleCloudAccountCredentials() {
```

Comment:
no: https://github.com/spotify/docker-client/commit/b44708fc5bf78801c07849ea75410c1979264844

I just meant that I didn't think to add a factory like `ContainerRegistryAuthSupplier.forCredentials()`, all it has now is `forApplicationDefaultCredentials()` and `forStream(InputStream credentialsStream)`